config/settings.d/puppet.yml.example

```diff
@@ -1,10 +0,0 @@
----
```

Comment:
This file needs to stay since it's still used for the Puppet module. You just need to drop the provider part. In 3eaa9cc26ddcdb2c5dfd3f3b88447c661fc6da31 I moved the Puppet HTTP API provider into the module itself. This is a leftover from when there was a legacy provider. 7e7015aba068db541bd4f4cf1856bf69ebbe2953 removed that.